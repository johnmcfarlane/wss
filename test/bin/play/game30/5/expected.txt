 32  4,6  | HOESHIN
 28  4,7  | HESIONE
 24  4,8  | ESHIN
 24  4,8  | HSIEN
 22  4,7  | INSHOE
 20  1,8  - HEHE
 20  1,8  - HOHE
 19  3,8  - HEE
 19  7,1  | HEHE
 19  9,1  | HEHE
 19  3,8  - HIE
 19  3,8  - HOE
 19  7,1  | HOHE
 19  9,1  | HOHE
 18  4,7  | EOSINE
 18  2,10 - HEH
 18  5,7  - HI
 18  7,1  | HINE
 18  9,1  | HINE
 18  1,8  - HINE
 18  7,1  | HONE
 18  9,1  | HONE
 18  1,8  - HONE
 17  4,10 - HEHE
 17  4,7  - HEI
 17  4,10 - HOHE
 17  4,7  - HOI
 17  3,8  - NEE
 17  3,8  - NEO
 17  4,7  - ONI
 16  9,3  | HEII
 16  4,7  | HISH
 16 10,2  | HONES
 16  4,5  | HONES
 16  4,7  | HOSEN
 16  4,7  - NEI
 16  2,10 - NOH
 15  8,4  - FOEHN
 14  2,8  - HEE
 14  2,8  - HEI
 14  4,6  | HENS
 14  2,8  - HIE
 14  4,6  | HINS
 14  4,7  | HISN
 14  2,8  - HOE
 14  2,8  - HOI
 14  4,6  | HOISE
 14  1,8  - IONE
 14  4,7  | NESH
 14  4,7  | NOSH
 14  5,7  - OI
 14  4,9  | SHIH
 13  4,8  - EE
 13  4,8  - EO
 13  4,8  - IO
 13  4,8  - OE
 13  4,9  | SINH
 12 10,8  | DOH
 12  4,7  | EOSIN
 12  3,8  - HE
 12  2,10 - HEI
 12  3,8  - HI
 12  4,6  | HIES
 12  3,8  - HO
 12  4,6  | HOES
 12  2,10 - HOI
 12  4,7  | HOSE
 12  4,7  | INSEE
 12  7,1  | IONE
 12  9,1  | IONE
 12  8,5  - LOIN
 12  2,8  - NEE
 12  2,8  - NEI
 12  2,8  - NEO
 12  4,6  | NOISE
 12  2,8  - ONE
 12  2,8  - ONI
 12  4,9  | SHINE
 12  4,9  | SHONE
 11 10,8  | DHONI
 11 10,8  | DONEE
 11 10,7  - EH
 11  5,7  | HEW
 11  5,7  | HOW
 11 10,7  - OH
 11  6,6  | QI
 11  4,9  | SHEN
 11  4,9  | SHIN
 10 10,8  | DENE
 10 10,8  | DINE
 10 10,8  | DINO
 10 10,8  | DONE
 10 10,8  | DONI
 10 10,9  - EH
 10  4,6  | ENOS
 10  4,7  | ENSE
 10  4,6  | EONS
 10  4,8  | ESNE
 10  8,4  - FOHN
 10  7,2  | HEE
 10  9,2  | HEE
 10  7,2  | HEI
 10  9,2  | HEI
 10  4,7  | HES
 10  7,2  | HIE
 10  9,2  | HIE
 10  4,7  | HIS
 10  7,2  | HOE
 10  9,2  | HOE
 10  7,2  | HOI
 10  9,2  | HOI
 10  4,8  | HSI
 10  4,7  | IHS
 10 10,7  - IONE
 10  4,6  | IONS
 10  4,8  | ISH
 10  8,5  - LOE
 10  3,8  - NE
 10  2,10 - NEI
 10  2,10 - NEO
 10  4,7  | NESE
 10  4,6  | NIES
 10  3,8  - NO
 10  4,6  | NOES
 10  4,7  | NOSE
 10 10,9  - OH
 10  4,7  | OHS
 10 10,7  - ONE
 10  4,6  | ONES
 10 10,7  - ONI
 10  2,10 - ONI
 10  4,9  | SEINE
 10  4,9  | SHEEN
 10 10,6  | SIDHE
  9 10,8  | DEINO
  9 10,8  | DEN
  9 10,8  | DIENE
  9 10,8  | DIN
  9 10,8  | DIONE
  9 10,8  | DON
  9 10,7  - EEN
  9 10,7  - EN
  9 10,7  - EON
  9  7,3  | HE
  9  9,3  | HE
  9  7,3  | HI
  9  9,3  | HI
  9  7,3  | HO
  9  5,11 - HO
  9 10,3  | HOISED
  9 10,7  - IN
  9 10,7  - ION
  9 10,9  - IONE
  9  7,2  | NEE
  9  9,2  | NEE
  9  7,2  | NEI
  9  9,2  | NEI
  9  7,2  | NEO
  9  4,11 - NEO
  9  5,7  | NEW
  9  5,7  | NOW
  9 10,7  - ON
  9  7,2  | ONE
  9  9,2  | ONE
  9 10,9  - ONE
  9  7,2  | ONI
  9  9,2  | ONI
  9 10,9  - ONI
  9  4,9  | SEEN
  9  4,9  | SION
  8 10,8  | DION
  8  3,8  - EE
  8 10,9  - EEN
  8  3,10 - EH
  8 10,9  - EN
  8  5,4  - ENIF
  8  4,7  | ENS
  8  3,8  - EO
  8 10,9  - EON
  8  8,4  - FEH
  8  8,4  - FINE
  8  8,4  - FINO
  8  8,4  - FOH
  8  8,4  - FOIN
  8  8,4  - FONE
  8 10,4  | HOSED
  8 10,9  - IN
  8  6,4  - INFO
  8  4,7  | INS
  8  3,8  - IO
  8 10,9  - ION
  8 10,5  - IONE
  8  4,8  | ISN
  8  7,3  | NE
  8  9,3  | NE
  8  5,4  - NEIF
  8  6,4  - NIFE
  8  4,7  | NIS
  8  7,3  | NO
  8  5,11 - NO
  8 10,3  | NOISED
  8  4,7  | NOS
  8  3,8  - OE
  8  3,10 - OH
  8  6,9  | OHONE
  8  3,8  - OI
  8 10,9  - ON
  8  4,7  | ONS
  8 10,4  | ONSIDE
  8  4,9  | SHEE
  8  4,9  | SHOE
  7  7,3  | EE
  7  9,3  | EE
  7 10,7  - EE
  7  7,3  | EO
  7 10,7  - EO
  7  5,11 - EO
  7 10,4  | EOSIDE
  7  8,4  - FEN
  7  8,4  - FIN
  7  8,4  - FON
  7 10,3  | HENS
  7 10,3  | HINS
  7  6,4  - INF
  7  7,3  | IO
  7 10,7  - IO
  7  5,11 - IO
  7  8,5  - LO
  7  6,4  - NEF
  7 10,4  | NOSED
  7  7,3  | OE
  7  9,3  | OE
  7 10,7  - OE
  7  7,3  | OI
  7  9,3  | OI
  7  9,5  | OI
  7 10,7  - OI
  7  4,9  | SHH
  7  4,9  | SINE
  7  4,9  | SNEE
  7  4,9  | SONE
  6 10,8  | DEE
  6 10,8  | DEI
  6 10,8  | DIE
  6 10,8  | DOE
  6 10,9  - EE
  6 10,5  - EEN
  6 10,5  - EH
  6 10,9  - EO
  6 10,5  - EON
  6  4,8  | ESE
  6  8,4  - FEE
  6  8,4  - FIE
  6  8,4  - FOE
  6  3,10 - HI
  6 10,3  | HIES
  6  3,10 - HO
  6 10,3  | HOES
  6  7,4  - IFE
  6 10,9  - IO
  6 10,5  - ION
  6  4,8  | ISE
  6 10,9  - OE
  6  4,7  | OES
  6 10,5  - OH
  6  6,9  | OHIO
  6 10,9  - OI
  6 10,5  - ONE
  6 10,5  - ONI
  6  4,8  | OSE
  6 10,5  | OSIDE
  5  7,4  - EF
  5 10,5  - EN
  5 10,3  | ENOS
  5 10,3  | EONS
  5  5,8  | EW
  5  8,4  - FE
  5  8,4  - FI
  5 10,4  | HES
  5 10,4  | HIS
  5  7,4  - IF
  5 10,4  | IHS
  5 10,5  - IN
  5 10,3  | IONS
  5 10,5  | ISED
  5 10,3  | NIES
  5  3,10 - NO
  5 10,3  | NOES
  5  7,4  - OF
  5  6,9  | OHO
  5 10,4  | OHS
  5 10,5  - ON
  5 10,3  | ONES
  5  5,8  | OW
  5  4,9  | SHE
  5  4,9  | SHI
  5  4,9  | SHO
  5 10,6  | SIDE
  5  4,9  | SOH
  4 10,5  - EE
  4 10,4  | ENS
  4 10,5  - EO
  4  3,10 - EO
  4  4,8  | ES
  4 10,4  | INS
  4 10,5  - IO
  4  3,10 - IO
  4  4,8  | IS
  4 10,4  | NIS
  4 10,4  | NOS
  4 10,5  - OE
  4 10,5  - OI
  4  3,10 - OI
  4 10,4  | ONS
  4  4,8  | OS
  4 10,6  | SED
  4  4,9  | SEN
  4  4,9  | SH
  4 10,6  | SID
  4  4,9  | SIN
  4 10,6  | SOD
  4  4,9  | SON
  3 10,8  | DE
  3 10,8  | DI
  3 10,8  | DO
  3 10,4  | OES
  3  4,9  | SEE
  3  4,9  | SEI
  3  4,9  | SIE
  3  4,9  | SOE
  2 10,5  | ES
  2 10,5  | IS
  2 10,5  | OS
  2  4,9  | SI
  2  4,9  | SO
