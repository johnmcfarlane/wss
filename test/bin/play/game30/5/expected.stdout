 18  4,7  | EOSINE
 18  2,10 - HEH
 18  5,7  - HI
 18  7,1  | HONE
 18  9,1  | HONE
 18  1,8  - HONE
 16  4,6  | HEHS
 16 10,2  | HONES
 16  4,5  | HONES
 16  4,7  | HOSEN
 16  2,10 - NOH
 15  8,4  - FOEHN
 14  4,6  | HENS
 14  2,8  - HIE
 14  4,6  | HINS
 14  4,7  | HISN
 14  2,8  - HOE
 14  4,6  | HOISE
 14  4,6  | HONS
 14  4,7  | NOSH
 14  5,7  - OI
 13  4,9  | SINH
 12  4,7  | EOSIN
 12  3,8  - HE
 12  3,8  - HI
 12  4,6  | HIES
 12  3,8  - HO
 12  4,6  | HOES
 12  4,7  | HOSE
 12  8,5  - LOIN
 12  2,8  - NEE
 12  4,6  | NOISE
 12  2,8  - ONE
 12  4,9  | SHINE
 12  4,9  | SHONE
 11 10,8  | DONEE
 11  5,7  | HEW
 11  5,7  | HOW
 11 10,7  - OH
 11  6,6  | QI
 11  4,9  | SHIN
 10 10,8  | DENE
 10 10,8  | DENI
 10 10,8  | DINE
 10 10,8  | DINO
 10 10,8  | DONE
 10 10,9  - EH
 10  4,6  | EONS
 10  4,8  | ESNE
 10  8,4  - FOHN
 10  4,7  | HES
 10  7,2  | HIE
 10  9,2  | HIE
 10  4,7  | HIS
 10  7,2  | HOE
 10  9,2  | HOE
 10  4,7  | HOS
 10  4,6  | IONS
 10  3,8  - NE
 10  3,8  - NO
 10  4,6  | NOES
 10  4,7  | NOSE
 10 10,9  - OH
 10  4,7  | OHS
 10 10,7  - ONE
 10  4,6  | ONES
 10  4,9  | SEINE
 10  4,9  | SHEEN
 10 10,6  | SIDHE
  9 10,8  | DEN
  9 10,8  | DIENE
  9 10,8  | DIN
  9 10,8  | DON
  9  7,3  | HE
  9  9,3  | HE
  9  7,3  | HI
  9  7,3  | HO
  9  5,11 - HO
  9 10,3  | HOISED
  9  7,2  | NEE
  9  9,2  | NEE
  9  5,7  | NEW
  9  5,7  | NOW
  9 10,7  - ON
  9  7,2  | ONE
  9  9,2  | ONE
  9 10,9  - ONE
  9  4,9  | SEEN
  8  3,10 - EH
  8 10,9  - EN
  8  4,7  | ENS
  8 10,9  - EON
  8  8,4  - FEH
  8  8,4  - FINE
  8  8,4  - FINO
  8  8,4  - FOH
  8  8,4  - FOIN
  8 10,3  | HEHS
  8 10,4  | HOSED
  8  6,4  - INFO
  8  4,7  | INS
  8  7,3  | NE
  8  9,3  | NE
  8  5,4  - NEIF
  8  7,3  | NO
  8  5,11 - NO
  8 10,3  | NOISED
  8  4,7  | NOS
  8  3,8  - OE
  8  3,10 - OH
  8  3,8  - OI
  8 10,9  - ON
  8  4,7  | ONS
  8 10,4  | ONSIDE
  8  4,9  | SHOE
  7  8,4  - FEN
  7  8,4  - FIN
  7  8,4  - FON
  7 10,3  | HENS
  7 10,3  | HINS
  7 10,3  | HONS
  7  8,5  - LO
  7 10,4  | NOSED
  7  7,3  | OE
  7  9,3  | OE
  7 10,7  - OE
  7  7,3  | OI
  7  9,5  | OI
  7 10,7  - OI
  7  4,9  | SENE
  7  4,9  | SHH
  7 10,6  | SIDH
  7  4,9  | SINE
  7  4,9  | SONE
  6 10,8  | DEE
  6 10,8  | DIE
  6 10,8  | DOE
  6 10,5  - EH
  6 10,5  - EON
  6  8,4  - FEE
  6  8,4  - FIE
  6  8,4  - FOE
  6  3,10 - HI
  6 10,3  | HIES
  6  3,10 - HO
  6 10,3  | HOES
  6 10,5  - ION
  6 10,9  - OE
  6  4,7  | OES
  6 10,5  - OH
  6 10,9  - OI
  6 10,5  - ONE
  6  4,8  | OSE
  5  7,4  - EF
  5 10,5  - EN
  5 10,3  | EONS
  5  8,4  - FE
  5 10,4  | HES
  5 10,4  | HIS
  5 10,4  | HOS
  5  7,4  - IF
  5 10,5  - IN
  5 10,3  | IONS
  5  3,10 - NO
  5 10,3  | NOES
  5  7,4  - OF
  5  6,9  | OHO
  5 10,4  | OHS
  5 10,5  - ON
  5 10,3  | ONES
  5  5,8  | OW
  5  4,9  | SHE
  5 10,6  | SIDE
  4 10,4  | ENS
  4  4,8  | ES
  4 10,4  | INS
  4  4,8  | IS
  4 10,4  | NOS
  4 10,5  - OE
  4 10,5  - OI
  4  3,10 - OI
  4 10,4  | ONS
  4  4,8  | OS
  4  4,9  | SEN
  4  4,9  | SH
  4  4,9  | SIN
  4 10,6  | SOD
  4  4,9  | SON
  3 10,8  | DE
  3 10,8  | DO
  3 10,4  | OES
  3  4,9  | SEE
  3  4,9  | SEI
  2 10,5  | ES
  2 10,5  | IS
  2 10,5  | OS
  2  4,9  | SI
  2  4,9  | SO
